src/background/datastore.js

```diff
@@ -25,6 +25,9 @@ export function convertInfo2Item(info) {
     id,
     title,
     origins,
+    created: info.timeCreated,
```

Comment:
nit: Might want to name these fields the same - e.g. `timeCreated: info.timeCreated`. We've got issue #21 to switch over to just matching Firefox's nsILoginInfo structure at some point, so it would probably be nice to have fewer things to rename